generics/helpers/solutionAndProjectTemplateUtils.js

```diff
@@ -394,8 +394,51 @@ function setScope(solutionId, scopeData, userDetails) {
 	})
 }
 
+/**
+ * Fetch related org details from user-service
+ * @method
+ * @name organizationDetails
+ * @param {Object} userDetails - loggedin user info
+ * @returns {Array} - Array of related org details
+ */
+function organizationDetails(userDetails) {
+	return new Promise(async (resolve, reject) => {
+		try {
+			// fetching related_org_details from user-service
+			let orgRead = await userService.fetchDefaultOrgDetails(userDetails.userInformation.organizations[0].id)
```

Comment:
@Prajwal17Tunerlabs check the changes @praveenKDass did for this user service call. he also renamed the function to "getOrgDetails". we should keep the same here as well